pkg/reconciler/knativeserving/ingress/gloo_test.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingress
+
+import (
+	"context"
+	"testing"
+
+	servingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+	util "knative.dev/operator/pkg/reconciler/common/testing"
+)
+
+func TestGlooTransformers(t *testing.T) {
+	instance := &servingv1alpha1.KnativeServing{}
+	transformer := glooTransformers(context.TODO(), instance)
+	util.AssertEqual(t, len(transformer), 0)
```

Comment:
This should fail if nil is returned from code... no?

Reply:
This test case is removed.